app/static/src/app/components/options/NumericInput.vue

```diff
@@ -69,10 +75,80 @@ export default defineComponent({
             }
 
             if (blur) {
-                errorTooltipProps.content = "";
+                // wrap tooltip props update with update to input box
+                // as the v-tooltip directive needs an update to actually
+                // hide the tooltip
+                textValue.value = `${textValue.value} `;
+                tooltipProps.value.content = "";
+                textValue.value = textValue.value.trim();
             }
         };
 
+        const validateArray = (arr: BoundTooltip[], numeric: number, ascending: boolean) => {
+            arr.sort((x, y) => {
+                if (x.number < y.number) return ascending ? -1 : 1;
+                if (x.number > y.number) return ascending ? 1 : -1;
+                return 0;
+            });
+
+            const condition = (compareNum: number) => {
+                return ascending ? numeric > compareNum : numeric < compareNum;
+            };
+
+            arr.forEach((bound) => {
```

Comment:
This seems odd as a `forEach` - if we set the tooltip multiple times if will just overwrite the setting - do we want to just use the first (or last?) triggered condition, if any?

Reply:
outdated